ank/src/cli_commands.rs

```diff
@@ -823,16 +906,125 @@ impl CliCommands {
             .insert(workload_name, new_workload);
 
         let update_mask = vec!["desiredState".to_string()];
+
+        self.internal_set_state(new_state, update_mask).await
+    }
+
+    async fn internal_set_state(
```

Comment:
I am not sure if this name describes what it does. It is called in several functions. My understanding is that it is a kind of "output_wait_list".

Reply:
I have renamed this function to `update_state_and_wait_for_complete`.